src/styles/designTokens.js

```diff
@@ -35,9 +35,10 @@ export const fontLarge = '1.8rem';
 export const fontXLarge = '2.4rem';
 
 // Font weight
+export const fontWeightThin = 300;
 export const fontWeightNormal = 400;
+export const fontWeightMedium = 500;
```

Comment:
Do we really need yet another font weight? We are using system font so it varies on devices / OSes anyway.